Basic FE project template/index.html

```diff
@@ -84,6 +94,81 @@ <h5 class="title-text">Possibility of return</h5>
           </div>
         </div>
       </section>
+      <section class="renting">
+        <div class="container">
+          <div class="renting__container">
+            <div class="renting__small-container">
+              <div class="renting__title">
+                <h1>RENTING IS PURE CONVENIENCE</h1>
```

Comment:
text should never be written in uppercase, write it normally, give it a class, and usee scss for it, or style the h1 globally to be uppercase